graphwebhook/src/main/resources/META-INF/additional-spring-configuration-metadata.json

```diff
@@ -0,0 +1,34 @@
+{
+  "properties": [
+    {
+      "name": "azure.activedirectory.keydiscoveryurl",
```

Comment:
should the app id/secret and tenant id be here as well?

Reply:
the Azure starter adds them - the only things added here are my own "custom" entries (otherwise dev tools complain about unknown values)